integration/spark/src/main/scala/org/apache/carbondata/spark/load/DataLoadProcessBuilderOnSpark.scala

```diff
@@ -342,7 +347,12 @@ object DataLoadProcessBuilderOnSpark {
       .mapPartitionsWithIndex { case (index, rows) =>
         ThreadLocalSessionInfo.setConfigurationToCurrentThread(conf.value.value)
         DataLoadProcessorStepOnSpark
-          .convertFunc(rows, index, modelBroadcast, partialSuccessAccum, convertStepRowCounter)
+          .convertFunc(rows,
+            index,
+            modelBroadcast,
+            true,
```

Comment:
this flow looks like load flow. This change might not be needed. can test and remove if not required

Reply:
done